src/xcode/ENA/ENA/Source/Client/__tests__/Mocks/ClientMock.swift

```diff
@@ -59,7 +63,13 @@ extension ClientMock: Client {
 	}
 
 	func getTestResult(forDevice device: String, completion completeWith: @escaping TestResultHandler) {
-		completeWith(.success(2))
+		guard let onGetTestResult = self.onGetTestResult else {
+			let positiveTestResult = 2
```

Comment:
would it perhaps be more clean / easier if you introduced an enum for these test results? you can put it in the top of the class.

Reply:
agree, I replaced it.